mysql/datadog_checks/mysql/version_utils.py

```diff
@@ -20,7 +20,10 @@ def get_version(db):
         # See http://dev.mysql.com/doc/refman/4.1/en/information-functions.html#function_version
         # https://mariadb.com/kb/en/library/version/
         # and https://mariadb.com/kb/en/library/server-system-variables/#version
-        parts = result[0].split('-')
+        raw_version = result[0]
+        if isinstance(raw_version, bytes):
+            raw_version = raw_version.decode()
+        parts = raw_version.split('-')
```

Comment:
raw_version being bytes is expected at least in python2.
We can ensure that the string is of the right type using `to_native_string`:
```suggestion
        raw_version = to_native_string(result[0])
        parts = raw_version.split('-')
```

Reply:
Didn't know we had that 👍